doc/dev/developer_guide/debugging-gdb.rst

```diff
@@ -0,0 +1,32 @@
+GDB - The GNU Project Debugger
+==============================
+
+`The GNU Project Debugger (GDB) <https://www.sourceware.org/gdb>`_ is
+a powerful tool which allows you to analyze the execution flow.
```

Comment:
Just realizing now that this sentence kind of halts early. Picky suggestion, but I think adding "of a ___" helps improve the clarity.
```suggestion
a powerful tool that allows you to analyze the execution flow of a process.
```

Reply:
Either that, or simply remove "the".
```suggestion
a powerful tool that allows you to analyze execution flow.
```